src/notification/publish/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "publish",
+  "version": "1.0.0",
+  "description": "",
+  "main": "publish.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "publish": "npx ts-node publish.ts"
```

Comment:
@KevinLemon112 The reason why the publish notification service wasn't working previously was there was no `npx ts-node` that is required to compile and run typescript code.